blogposts/governance-app.md

```diff
@@ -0,0 +1,81 @@
+blockchain governance.
+
+The Governance App was envisioned as a tool to empower the aeternity community, enabling them to have a say in the
+platform's direction through a transparent and verifiable polling mechanism. The Unit 214 Team was commissioned to
+implement this vision. The development process was focused on their commitment to open-source principles, with actively
+seeking and incorporating feedback from testing phases to refine and improve the app. Key challenges such as user
+accessibility and poll result verification were addressed through iterative testing and community feedback.
+
+## ⚙️ Features and Functionality
+
+At its core, the Governance App allows users to create and participate in polls, facilitating a democratic
+decision-making process within the aeternity ecosystem. It emphasises ease of use, with a user-friendly interface that
+makes participation accessible to all members of the community. To ensure transparency and trust, the app features
+robust verification mechanisms for poll results, and ongoing optimizations have significantly improved performance and
+user experience.
+
+## 🛠️ Challenges and Solutions
+
+The development of the aeternity Governance App had three core technical challenges, each addressed through innovative
+use of technology and programming frameworks:
+
+### 🔄 Liquid Democracy and Real-time Aggregation
+
+Implementing a liquid democracy system, with its potential for infinite delegations, required a robust and fast solution
+for real-time vote tallying. We leveraged smart contracts on the blockchain to manage the complex logic of delegations
+and secure voting. The Node.js backend, developed with efficiency in mind, ensures accurate, transparent, and immediate
+aggregation of polling results, embodying the democratic ideals of the app.
+
+### 🌍 Application Optimization for Global Accessibility
+
+To enable the Governance App's responsiveness worldwide, especially on slower networks, we focused on minimising the
+application's size. Utilising Vue.js for the frontend allows for a lightweight, yet dynamic user interface. On the
+backend, Node.js, combined with efficient database management through PostgreSQL and caching solutions via Redis,
+significantly enhances the app's delivery speeds and overall performance.
+
+### 💪 Resilience and Independence
+
+Building an app capable of functioning independently of its backend was a priority, ensuring usability even when the
+backend was offline or not trustworthy. Smart contracts play a pivotal role in maintaining functionality, allowing users
+to interact with the blockchain directly and securely for critical operations like voting and delegations. Additionally,
+the app is also validating the smart contracts code to ensure their integrity.
+
+These challenges were met with a blend of modern technologies and innovative approaches, ensuring the Governance App was
```

Comment:
Would it be possible to elaborate on the "modern technologies" used and "innovative approaches" applied?